niworkflows/func/util.py

```diff
@@ -109,14 +111,15 @@ def init_bold_reference_wf(omp_nthreads, bold_file=None, pre_mask=False,
     outputnode = pe.Node(
         niu.IdentityInterface(fields=['bold_file', 'raw_ref_image', 'skip_vols',
                                       'algo_dummy_scans', 'ref_image', 'ref_image_brain',
-                                      'bold_mask', 'validation_report', 'mask _report']),
+                                      'bold_mask', 'validation_report', 'mask_report']),
         name='outputnode')
 
     # Simplify manually setting input image
     if bold_file is not None:
         inputnode.inputs.bold_file = bold_file
 
-    validate = pe.Node(ValidateImage(), name='validate', mem_gb=DEFAULT_MEMORY_MIN_GB)
+    validate = pe.MapNode(ValidateImage(), name='validate', mem_gb=DEFAULT_MEMORY_MIN_GB,
```

Comment:
Could we force the developer to explicitly set a new ``multiecho=True`` argument to the pipeline to wrap the interface into a `MapNode` instead of a `Node`?

Reply:
Sorry, this was added in a04814f7c0c318194e17d651091cd436c2cca558 (and was fixed a couple of commits later).